README.md

```diff
@@ -54,6 +54,26 @@ call the method `Datadog::Statsd#flush` if you want to force the sending of metr
   * using singletons instances of the DogStatsD client and not allocating one each time you need one, letting the buffering mechanism flush metrics, or,
   * properly closing your DogStatsD client instance when it is not needed anymore using the method `Datadog::Statsd#close` to release the resources used by the instance and to close the socket
 
+If you have issues using the companion thread or the buffering mode, you can instantiate a client to behave exactly as it was in v4.x (flushing on every metric submission):
+
+```ruby
+# Import the library
+require 'datadog/statsd'
+
+# Create a DogStatsD client instance using UDP
+statsd = Datadog::Statsd.new('localhost', 8125, single_thread: true, buffer_max_payload_size: 1)
```

Comment:
we could explicitly mention below in this example that closing the instance is unnecessary (except to close the network socket), wdyt?

Ditto for the UDS example